settings/src/script.js

```diff
@@ -156,22 +165,15 @@ function Main( { userId } ) {
 	);
 
 	if ( 'account-status' === screen ) {
-		screenContent = (
-			<div className={ 'wporg-2fa__account-status' }>
-				<AccountStatus />
-			</div>
-		);
+		screenContent = <AccountStatus />;
 	} else if (
 		twoFactorRequiredScreens.includes( screen ) &&
-		userRecord.record[ '2fa_available_providers' ] &&
```

Comment:
🤦  This is JS.. Even an empty array is truthful here.. this should've had a `.length` appended.. that's half the reason for this PR :D  (No changes needed, just adding a 🤯 moment)